codeception.ci.functional.yml

```diff
@@ -2,7 +2,7 @@ namespace: PyzTest
 actor: Tester
 
 include:
-    - tests/PyzTest/*/*
+    - tests/PyzTest/Zed/*
```

Comment:
are you sure?

Reply:
taken from shop suite
but agree, this skips tests from Service/Client/Glue layers
will revert